src/components/Dashboard/Dashboard.js

```diff
@@ -41,37 +105,89 @@ export const Dashboard = () => {
         </div>
       </header>
       <div className="dp-container">
-        <S.Subtitle>
-          <span>
-            <img
-              src="https://cdn.fromdoppler.com/doppler-ui-library/latest/img/icon-book.svg"
-              alt="icon-title"
-            />
-          </span>
-          Aprende con Doppler
-        </S.Subtitle>
         <div className="dp-rowflex">
-          <div className="col-sm-12 col-md-6">
-            <Carousel id={'1'} color={'orange'}>
-              {({ activeSlide }) =>
-                postList.map((post, index) => (
-                  <Slide key={post.id} active={activeSlide === index}>
-                    <TextPreviewPost post={post} />
-                  </Slide>
-                ))
-              }
-            </Carousel>
+          <div className="col-lg-12 col-md-12 m-b-24">
+            <div className="dp-dashboard-title">
+              <DashboardIconSubTitle
+                title="Mis Campañas"
+                iconClass="deliveries"
+              ></DashboardIconSubTitle>
+              <DashboardIconLink linkTitle="CAMPAÑAS ENVIADAS" link="#"></DashboardIconLink>
+            </div>
+            <KpiGroup>
+              {kpiList
+                .filter((kpi) => kpi.kpiType === 'Campaings')
```

Comment:
this data should be in the `useEffect`. Because every time the Dashboard change the state, is executed the `filter` & the `map` functions